### ISSUE TITLE
Data partitioning

### ISSUE THREAD

**Issue by fabricecarles** [XuJiacong/PIDNet#64]
Hi, 
Could you explain why did you train on trainval.lst and validate accuracy with data already seen in training set (e.g val.lst) ? 

https://github.com/XuJiacong/PIDNet/blob/fefa51716bddc13a4321af2c70a074367100645a/configs/cityscapes/pidnet_large_cityscapes_trainval.yaml#L14C1-L15C44

Is it to train on overall data before testing on test.lst ? 

If yes I think you should replace 
```
  TEST_SET: 'list/cityscapes/val.lst'
  TRAIN_SET: 'list/cityscapes/trainval.lst'
```
by 
```
  TEST_SET: 'list/cityscapes/test.lst'
  TRAIN_SET: 'list/cityscapes/trainval.lst'
```
because your accuracy evaluation result in the current config is not correct from my understanding of the current code

**Comment by stemper0123** [comment]
That's a good point. I believe he used it for pretraining the model on CamVid, where accuracy is not as necessary. I couldn't find any information in his paper or his previous papers that he referenced when mentioning the fine-tuning of the cityscapes model on CamVid.

If you're considering using it to train on Cityscapes, just use the normal config without the _trainval suffix.
